src/MPAGSCipher/RunCaesarCipher.cpp

```diff
@@ -0,0 +1,34 @@
+#include <string>
+
+int positiveModulo(const int a, const int b){
+    /*
+    Calculates a % b such that if a < 0, the result is positive (0 < result < b-1)
+    */
+    return (a % b + b) % b;
+}
+
+std::string runCaesarCipher(const std::string& inputText, const size_t key, const bool encrypt){
+    /*
+    Encrypts/decrypts some text using a standard Caesar Cipher
+
+    const std::string& inputText: string of text to be encrypted/decrypted
+    const size_t key: integer in range [0, 25] - Caesar Cipher shift
+    const bool encrypt: if true, inputText will be shifted by 'key' to the right, or to the left if false.
+
+    return: string of encrypted/decrypted text
+    */
+    std::string cipherText{""};
+    int modifier{1};
+    
+    // If we want to decrypt, this modifier turns key -> -key
+    if (!encrypt){
+        modifier = -1;
+    }
+    // ASCII representations of capital letters range from A: 65 to Z: 90
+    // Add the key, subtract the ASCII offset, modulo 26 and re-add the offset
+    // This accounts for e.g. Z->A if key=1
+    for (char c : inputText){
+        cipherText += (char)(positiveModulo(c + modifier * key - 65, 26) + 65);
```

Comment:
Please avoid using C-style casts.  If you need to perform a cast it is preferable to use the named casts provided in C++, e.g.:
```suggestion
        cipherText += static_cast<char>(positiveModulo(c + modifier * key - 65, 26) + 65);
```
It is far easier to search for such cases if you ever need to modify them.  Also there are other sorts of casts available for different situations.
See the "Conversions" section of:
https://en.cppreference.com/w/cpp/language/expressions
and
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-casts
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-casts-named